common/lib/capa/capa/xqueue_interface.py

```diff
@@ -134,11 +134,15 @@ def _send_to_queue(self, header, body, files_to_upload):
 
     def _http_post(self, url, data, files=None):
         try:
-            r = self.session.post(url, data=data, files=files)
+            r = self.session.post(url, data=data, files=files, timeout=(3.05, 5))
```

Comment:
I've been looking through the traces and noticed there are some legit instances of it taking a few seconds, so let's up the read timeout to 10 seconds instead of 5.

Also, could you please put the connect timeout and read timeout as constants at the top of this module?